docker-compose.yml

```diff
@@ -46,16 +46,16 @@ services:
     restart: always
     ports:
     - "40007:3000"
-  agar-io:
-    build: agar-io
-    restart: always
-    ports:
-    - "40008:3000"
+  #agar-io:
+  #  build: agar-io
+  #  restart: always
+  #  ports:
+  #  - "40008:3000"
```

Comment:
Doesn't the agar.io Dockerfile work?
Otherwise I don't think we should comment it out.